# Housing Market

### Introduction:

This time we will create our own dataset with fictional numbers to describe a house market. As we are going to create random data don't try to reason of the numbers.

### Step 1. Import the necessary libraries

In [1]:
import pandas as pd
import numpy as np

### Step 2. Create 3 differents Series, each of length 100, as follows:
1. The first a random number from 1 to 4
2. The second a random number from 1 to 3
3. The third a random number from 10,000 to 30,000

In [2]:
s1 = pd.Series(np.random.randint(1, 5, 100))
s2 = pd.Series(np.random.randint(1, 4, 100))
s3 = pd.Series(np.random.randint(10000, 30001, 100))

### Step 3. Let's create a DataFrame by joinning the Series by column

In [3]:
df = pd.concat([s1, s2, s3], axis=1)

### Step 4. Change the name of the columns to bedrs, bathrs, price_sqr_meter

In [4]:
df.columns = ['bedrs', 'bathrs', 'price_sqr_meter']

### Step 5. Create a one column DataFrame with the values of the 3 Series and assign it to 'bigcolumn'

In [5]:
bigcolumn = pd.concat([s1, s2, s3]).reset_index(drop=True)


### Step 6. Oops, it seems it is going only until index 99. Is it true?

In [6]:
print(bigcolumn.index.max())

299


### Step 7. Reindex the DataFrame so it goes from 0 to 299

In [7]:
bigcolumn = bigcolumn.reindex(range(300))


In [8]:
print(df.head())
print(bigcolumn.tail())

   bedrs  bathrs  price_sqr_meter
0      2       3            23451
1      4       1            18291
2      2       3            22235
3      4       3            27882
4      3       3            22035
295    16268
296    15483
297    12022
298    14681
299    18136
dtype: int64
